.github/workflows/dev-smoketest.yaml

```diff
@@ -0,0 +1,19 @@
+name: dev-smoketest
+on:
+  push:
+    branches:
+      ci-testflight-github-action
+jobs:
+  smoketest:
+    name: Hit smoketests against Charts in Dev Setup
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install K3D
+        run: wget -q -O - https://raw.githubusercontent.com/k3d-io/k3d/main/install.sh | bash
```

Comment:
I think this will need terraform installed as well as it is not included with k3d

```
wget -O- https://apt.releases.hashicorp.com/gpg | gpg --dearmor | sudo tee /usr/share/keyrings/hashicorp-archive-keyring.gpg

echo "deb [signed-by=/usr/share/keyrings/hashicorp-archive-keyring.gpg] https://apt.releases.hashicorp.com $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/hashicorp.list
sudo apt update && sudo apt install terraform
```

Reply:
Terraform is installed by default on GitHub Actions Image.

https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2204-Readme.md#tools

Reply:
yeah, I see now the workflow is running anyway